### PR TITLE
Adding in locker information to assignment email

### LINKED ISSUE #8: Email the patron when a locker is assigned
Here are examples fo the current report:
https://docs.google.com/document/d/1GUEzMUo-dKjok0tjhkQcs_xCFW-RivonobhM3PLED6g/edit#heading=h.p74uv6pdgcz

Feedback on the current process is here (item 3): 
https://docs.google.com/document/d/11EK-yBRfOYytMzTR3LxtoypAIjX0n_Yan4cHeMrFXpU/edit

### PR DESCRIPTION
**Pull request by carolyncole**
Also added agreement as attachment to the email.

fixes #8

### CODE REVIEW

File: app/views/user_mailer/locker_assignment_confirmation.text.erb [hunk 1]
```diff
@@ -1,4 +1,12 @@
...
 <%= @locker_assignment.name %> is assigned to <%= @locker_assignment.location %>
 Occupency expires on <%= @locker_assignment.expiration_date %>
-Applied as <%= @locker_assignment.status_at_application %> in the <%= @locker_assignment.department %> Department
\ No newline at end of file
+Applied as <%= @locker_assignment.status_at_application %> in the <%= @locker_assignment.department %> Department
+Lock Combination: <%= @locker_assignment.combination %>
+To Unlock: 
+1. Turn knob RIGHT three time and stop at the first number
+2. Turn knob LEFT one full tuen passing the first number in the combination stopping at the second number in the cobination.
+3. Turn knob TIGHT to the last number in the combination. Continue turning until the dial stops.
```
Comment (christinach) [inline app/views/user_mailer/locker_assignment_confirmation.text.erb:9]: tuen -> then

Comment (christinach) [reply app/views/user_mailer/locker_assignment_confirmation.text.erb:9]: is this TIGHT or RIGHT ?

File: spec/mailers/user_spec.rb [hunk 2]
```diff
@@ -11,8 +11,13 @@
...
       expect(mail.subject).to eq 'Your Locker has been assigned'
       expect(mail.to).to eq [locker_assignment.email]
+      expect(mail.from).to eq ['access@princeton.edu']
       expect(mail.html_part.body.to_s).to have_content('Firestone Library Locker Assignment')
       expect(mail.html_part.body.to_s).to have_content(locker_assignment.name)
+      expect(mail.html_part.body.to_s).to have_content(locker_assignment.combination)
+      expect(mail.html_part.body.to_s).to have_content('To Unlock')
+      expect(mail.html_part.body.to_s).to have_content('To Lock')
+      expect(mail.attachments.first.content_type).to eq('application/pdf; filename="Locker Space Agreement.pdf"')
     end
```
Comment (christinach) [inline spec/mailers/user_spec.rb:21]: 👍🏻

File: app/views/user_mailer/locker_assignment_confirmation.text.erb (hunk 1 above)
Comment (christinach) [reply app/views/user_mailer/locker_assignment_confirmation.text.erb:9]: @carolyncole ^

Comment (carolyncole) [reply app/views/user_mailer/locker_assignment_confirmation.text.erb:9]: @christinach good catch.  Thanks!

**Comment by carolyncole** [comment]
@christinach Thanks!  I also found cobination in the text and updated that.

**Review by christinach — APPROVED** [review]

**Pull request merged**